src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -36,3 +38,39 @@ The following experimental features are available in New Relic Browser:
   } }:
   ```
 4. Deploy your app.
+
+### NPM Implementations
+
+#### Browser Performance - Marks, Measures & Resources
```

Comment:
With same explanation above:

###NPM Implementation of Marks, Measures, or Page resources

Reply:
https://github.com/newrelic/docs-website/pull/19522#discussion_r1886986442